client/components/Menu/components/Navigation/Navigation.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import './Navigation.scss';
+import AddIco from 'images/add.svg';
+import UserMenuIco from 'images/user-menu.svg';
+import EnvelopeIco from 'images/envelope.svg';
+import DashbordIco from 'images/dashbord.svg';
+
+const MENU_ITEM = [
+  { name: 'MY PROFILE', path: '/profile', ico: UserMenuIco },
+  { name: 'CONTACT', path: '/contacts', ico: EnvelopeIco },
+  { name: 'DASHBORD', path: '/dashbord', ico: DashbordIco }
+];
+
+const NavigationItem = ({ name, path, ico }) => (
+  <li>
+    <Link className="navigation_item" to={path}>
+      <img src={ico} alt="ico" />
+      {name}
+    </Link>
+  </li>
+);
+
+const Navigation = ({ sidebar }) => (
+  <nav className="navigation">
+    <ul>
+      <li>
+        <Link className="navigation_item" to="/" onClick={sidebar}>
+          <img src={AddIco} alt="add" />
+          NEW TRIP
```

Comment:
You can use lover case and handle upper case with styles

Reply:
Fixed